kategory-effects/src/main/kotlin/kategory/effects/instances/JobWInstances.kt

```diff
@@ -0,0 +1,44 @@
+package kategory
+
+import kotlin.coroutines.experimental.CoroutineContext
+
+interface JobWInstances :
```

Comment:
:fire:

Reply:
It cannot be derived because several methods depend on CoroutineContext, and MonadError has a preapplied Error type.